internal/integration/fixtures/extra-volume-mounts.yaml

```diff
@@ -0,0 +1,27 @@
+steps:
+  - label: ":wave:"
+    agents:
+      queue: {{.queue}}
+    env:
+      BUILDKITE_SHELL: /bin/sh -e -c
```

Comment:
```suggestion
```
we don't need this anymore. It used to be necessary for alpine containers, (and it still is), but now, the controller injects this by default: https://github.com/kingatlas/agent-stack-k8s/blob/878c49c75a3af0cb4b0c8ccca1232409e820b42f/internal/controller/scheduler/scheduler.go#L258-L261

It's still around in older integration tests though, but that's due to inertia.